### DISCUSSION TITLE
Adblockid Plus - dapat digunakan untuk memblokir situs dewasa?

### DISCUSSION THREAD

**Discussion by arfshl** [realodix/AdBlockID#1140] [Q&A] [answered]
Apakah adblockid plus dapat digunakan untuk memblokir situs dewasa

**Comment by realodix** [comment, marked as answer]
Tidak bisa.

Sama seperti AdBlockID, sifatnya AdBlockID Plus hanya memblokir iklannya saja. Perbedaan paling mencolok adalah AdBlockID Plus dapat dimasukkan domain situs dewasa untuk diblokir iklannya secara manual.

AdBlockID - Iklan di situs dewasa **kemungkinan masih ada**.
AdBlockID Plus - Karena situs dewasa dapat masuk, iklan yang ada di situs dewasa **diusahakan hilang**.

---

Mungkin ke depan bisa dibuat produk baru misalkan "AdBlockID+ (Porn Block Host)" khusus untuk blokir akses situs dewasa.

**Reply by arfshl** [reply]
:)

**Reply by Jokopentil** [reply]
Menurut aku gak usah bikin seperti itu
Klo hanya blok situs porno bisa pakai cleanbrowsing dns
Yang ada di fitur chrome
Pakai saja yang cleanbrowsing family filter
Justru yang perlu itu blockir iklan judi dan situs dating porn chat itu sarangnya pop up dan virus

**Comment by 3xploiton3** [comment]
kalau mau blokir konten dewasa dipengaturan google pilih safebrowsing, atau gunakan dns family contoh adguard

https://adguard-dns.io/kb/general/dns-providers/
